src/wire/ip.rs

```diff
@@ -184,6 +184,23 @@ impl Address {
             &Address::__Nonexhaustive => unreachable!()
         }
     }
+
+    /// Treat as subnet mask, get position of first 0
+    pub fn to_prefix_len(&self) -> u8 {
```

Comment:
Either this function shouldn't be `pub`, or it should perform more validation and ensure that `self` is a CIDR subnet mask.

Reply:
I pushed another commit now that adds `to_prefix_len()` with the requested validation. Because the fn can remain pub, Travis-CI should run without any unused fn failures this time.